gradle/versions.gradle

```diff
@@ -52,7 +52,7 @@ dependencyManagement {
 
     dependency 'org.mock-server:mockserver-junit-jupiter:5.11.2'
 
-    dependencySet(group: 'io.swagger.core.v3', version: '2.1.11') {
+    dependencySet(group: 'io.swagger.core.v3', version: '2.1.12') {
```

Comment:
Just be careful that swagger still renders, often its a requirement from javalin that there's a specific swagger, so updating this may break it...
I haven't checked this one specifically but in the past it'l stop rendering if we go and change it...

Reply:
I checked mavencentral and it looks like it should be ok... I'd love a way to fail a test if it fails... will have a think

Reply:
Yep, confirmed it still renders.